source/octf/node/NodeBase.cpp

```diff
@@ -41,16 +41,22 @@ void NodeBase::deinitCommon() {
 
 bool NodeBase::readSettings() {
     if (m_config) {
-        ProtobufReaderWriter rw(
-                getFrameworkConfiguration().getNodeSettingsFilePath(
-                        getNodePath()));
+        try {
```

Comment:
Impossibility of reading previous settings it's critical, so I'd throw exception to stop initialization immediately.